shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -667,53 +613,68 @@ public int run(String[] argv) {
       return -1;
     }
 
+    // Sanity check on the number of arguments
     String cmd = argv[0];
+    int numOfArgs = getNumOfArgs(cmd);    
+    if (numOfArgs != argv.length) {
+      System.out.println(cmd + " takes " + numOfArgs + " parameters.\n\n");
```

Comment:
oh, I'm stupid. I've already used System.out.println(). I will change to one "\n"